components.yaml

```diff
@@ -1,22 +1,29 @@
 components:
-  multus:
+  - name: "multus"
     url: "https://github.com/intel/multus-cni"
-    commit: "bfaf22964b51b93b08dcb1f10cbd8f8cb9a3603a" # v3.4.1
-  linux-bridge:
+    commit: "bfaf22964b51b93b08dcb1f10cbd8f8cb9a3603a"
+    release_tag: "v3.4.1"
```

Comment:
Thanks for keeping the tag somewhere, I know it can remain as a comment.

Is it a good name though? What if we set our strategy to always take the latest available code on a branch? Maybe a more generic name to keep this metadata would be suitable? It is only for our eyes, not to be processed.

Reply:
I agree. I think is good for humans to look at the yaml and know exactly what it holds. The only way I can modify it properly it is to add it as part of the struct.
Will change to Metadata.